index.html

```diff
@@ -133,6 +133,190 @@ <h2 id="headline-text">I'm Kanza. Glad to see you!</h2>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+                <button class="project-btn">See Project</button>
+              </div>
+            </div>
+          </div>
+          <div class="card">
+            <div class="inner-card">
+              <div>
+                <img
+                  src="./images/second-section/Snapshoot-04.png"
+                  alt="Snapshoot"
+                />
+              </div>
+              <div class="desc">
+                <h3>Tonic</h3>
+                <h4>Canopy <span>Back End Dev</span> <span>2015</span></h4>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+                </ul>
+                <button class="project-btn">See Project</button>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section id="about-me">
+          <h2 class="headline-text">About Myself</h2>
+          <p class="supporting-text">
+            Hello I'm a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+          </p>
+          <button type="button" class="connect">LET'S CONNECT</button>
+          <div class="social-icons">
+            <a
+              title="link"
+              class="icons"
+              href="https://github.com/KanzaTahreem"
+              target="_blank"
+              rel="noopener"
+              ><i class="fa-brands fa-github"></i
+            ></a>
+            <a
+              title="link"
+              class="icons"
+              href="https://www.linkedin.com/in/kanza-tahreem/"
+              target="_blank"
+              rel="noopener"
+              ><i class="fa-brands fa-linkedin-in"></i
+            ></a>
+            <a
+              title="link"
+              class="icons"
+              href="#"
+              target="_blank"
+              rel="noopener"
+              ><i class="fa-brands fa-angellist"></i
+            ></a>
+            <a
+              title="link"
+              class="icons"
+              href="https://twitter.com/kanza_tahreem"
+              target="_blank"
+              rel="noopener"
+            >
+              <i class="fa-brands fa-twitter"></i
+            ></a>
+            <a
+              title="link"
+              class="icons"
+              href="mailto:tehreemmsk411@gmail.com"
```

Comment:
Kindly, I suggest that the icon listings use UL/LI lists, your site will be better structured